home-manager/linux.nix

```diff
@@ -36,6 +36,9 @@ lib.mkMerge [
         podman-tui
         docker-compose
 
+        kubectl
+        kind
```

Comment:
🤔 Globally put kubectl is good for that purpose. However kind might be only required in sandbox repositories